packages/codebytes/src/index.tsx

```diff
@@ -81,14 +89,14 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
           onChange={(newText: string) => {
             setText(newText);
             onEdit?.(newText, language);
-
             const { renderMode } = getOptions();
             if (!renderMode && hasBeenEdited === false) {
               setHasBeenEdited(true);
               trackClick('edit');
             }
           }}
           snippetsBaseUrl={snippetsBaseUrl}
+          onCopy={onCopy}
```

Comment:
did you mean to add oncopy here? was this related to a behavior we didn't realize until we added tests?

Reply:
@borisonr `onCopy` is intended to be here! we had it in the original code, and it may have gotten removed with the tracking updates, but we should see it trickle down to the Editor component as we had in previous prs. 

when called in the `Editor` component, we can call it with the specific `onCopy` behavior that's needed for the iframe version or for the homepage version.

Reply:
glad we caught it!

Reply:
@borisonr thank you for your ever-amazing reviews!